pkg/components/velero/component_test.go

```diff
@@ -208,7 +208,7 @@ component "velero" {
       bucket   = "foo"
       provider = "aws"
     }
-		
```

Comment:
Seems unrelated.

Reply:
Yes, its just whitespace so I think can be safely ignored?